Package.swift

```diff
@@ -36,6 +36,11 @@ let package = Package(
     ]
 )
 
+#if os(Linux)
+package.targets.append(.systemLibrary(name: "playerctl", pkgConfig: "playerctl"))
```

Comment:
you can use conditional dependency, no compilation condition needed.
```swift
.target(name: "playerctl", condition: .when(platforms: [.linux]))
```

Reply:
Cannot build on macOS when using conditional target:
> error: Source files for target playerctl should be located under 'Sources/playerctl'

and the "systemLibrary" has no condition.

Reply:
It works to me (on macOS):
```swift
.target(
    name: "MusicPlayer",
    dependencies: [
        .target(name: "playerctl", condition: .when(platforms: [.linux])),
        ...
    ]),
.systemLibrary(name: "playerctl", pkgConfig: "playerctl"),
```

SPM won't build unused library `playerctl` on macOS.